[2024-09-07T15:50:02.456+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-09-07T15:50:02.606+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: example_postgres_pandas_dag_with_decorators.process_data manual__2024-09-07T15:49:51.502093+00:00 [queued]>
[2024-09-07T15:50:02.627+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: example_postgres_pandas_dag_with_decorators.process_data manual__2024-09-07T15:49:51.502093+00:00 [queued]>
[2024-09-07T15:50:02.628+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-09-07T15:50:02.685+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): process_data> on 2024-09-07 15:49:51.502093+00:00
[2024-09-07T15:50:02.715+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'example_postgres_pandas_dag_with_decorators', 'process_data', 'manual__2024-09-07T15:49:51.502093+00:00', '--job-id', '61', '--raw', '--subdir', 'DAGS_FOLDER/etl_prueba.py', '--cfg-path', '/tmp/tmpcktkxsu6']
[2024-09-07T15:50:02.718+0000] {standard_task_runner.py:91} INFO - Job 61: Subtask process_data
[2024-09-07T15:50:02.730+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=381) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-07T15:50:02.732+0000] {standard_task_runner.py:63} INFO - Started process 383 to run task
[2024-09-07T15:50:02.834+0000] {task_command.py:426} INFO - Running <TaskInstance: example_postgres_pandas_dag_with_decorators.process_data manual__2024-09-07T15:49:51.502093+00:00 [running]> on host 9debaee6dbd1
[2024-09-07T15:50:03.405+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='example_postgres_pandas_dag_with_decorators' AIRFLOW_CTX_TASK_ID='process_data' AIRFLOW_CTX_EXECUTION_DATE='2024-09-07T15:49:51.502093+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-07T15:49:51.502093+00:00'
[2024-09-07T15:50:03.410+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-09-07T15:50:03.432+0000] {base.py:84} INFO - Using connection ID 'postgres' for task execution.
[2024-09-07T15:50:03.477+0000] {logging_mixin.py:188} WARNING - /opt/***/dags/etl_prueba.py:35 UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
[2024-09-07T15:50:03.497+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-09-07T15:50:03.498+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-09-07T15:50:03.514+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=example_postgres_pandas_dag_with_decorators, task_id=process_data, run_id=manual__2024-09-07T15:49:51.502093+00:00, execution_date=20240907T154951, start_date=20240907T155002, end_date=20240907T155003
[2024-09-07T15:50:03.659+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-09-07T15:50:03.722+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-07T15:50:03.735+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
